src/styles/blocks/link.css

```diff
@@ -0,0 +1,23 @@
+.card__link {
```

Comment:
You need to sort your CSS properties.

[More info](https://codeguide.academy/html-css.html#css-order)

Reply:
okay